css-two-column-layout/styles.css

```diff
@@ -0,0 +1,87 @@
+img {
+  height: 125px;
+  width: auto;
+}
+
+button {
+  margin-top: 50px;
+  margin-left: 425px;
+  color: #d96a65;
+  border-color: #d96a65;
+  width: 150px;
+  height: 35px;
+  background-color: white;
+}
+
+body {
+  font-family: karla, sans-serif;
+}
+
+h1 {
+  margin-top: 40px;
+}
+
+h1,
+h3,
+button {
+  text-transform: uppercase;
+}
+
+.container {
+  max-width: 1000px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  width: 50%;
+}
+
+.textalignc {
+  text-align: center;
+}
+
+.full-column {
+  flex-basis: 100%;
+}
+
+.padding {
+  padding-top: 16px;
+  padding-bottom: 8px;
+}
+
+.lspace {
+  line-height: 2rem;
+}
+
+.paddingimg {
+  padding-right: 3px;
+}
+
+.column1-6 {
+  flex-basis: calc(100% / 6);
+}
+
+.column1-3 {
+  flex-basis: calc(100% / 3);
+  margin-right: 10%;
```

Comment:
`column` classes should only have `flex-basis` or `width` and nothing else. You should make an additional class you can add to the element in addition to the column class.